packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -1652,9 +1675,16 @@ class Route extends EmberObject implements IRoute {
     @public
   */
   render(name?: string, options?: PartialRenderOptions) {
-    let renderOptions = buildRenderOptions(this, name, options);
-    ROUTE_CONNECTIONS.get(this).push(renderOptions);
-    once(this._router, '_setOutlets');
+    deprecate('Usage of `render` is deprecated.', false, {
```

Comment:
Lets include some more information in the deprecation message. I'm thinking `this.routeName` at least...